formats/developers/compression.txt

```diff
@@ -0,0 +1,170 @@
+| OME-TIFF, 7-zipped        | Best – 108,692 bytes       | Poor – The appropriate OME-TIFF file must be uncompressed from the    |
+|                           |                            | archive.                                                              |
++---------------------------+----------------------------+-----------------------------------------------------------------------+
+| OME-TIFF, zipped          | Great – 122,031 bytes      | Poor – The appropriate OME-TIFF file must be uncompressed from the    |
+|                           |                            | archive.                                                              |
++---------------------------+----------------------------+-----------------------------------------------------------------------+
+| OME-TIFF+LZW, zipped      | Good – 134,834 bytes       | Poor – The appropriate OME-TIFF file must be uncompressed from the    |
+|                           |                            | archive, and the plane must be decoded from LZW.                      |
++---------------------------+----------------------------+-----------------------------------------------------------------------+
+| OME-TIFF+LZW, 7-zipped    | Good – 135,014 bytes       | Poor – The appropriate OME-TIFF file must be uncompressed from the    |
+|                           |                            | archive, and the plane must be decoded from LZW.                      |
++---------------------------+----------------------------+-----------------------------------------------------------------------+
+| OME-XML, gzipped          | Great – 124,763 bytes      | Worst – Entire dataset must be uncompressed, then the plane must be   |
+|                           |                            | decoded from base64.                                                  |
++---------------------------+----------------------------+-----------------------------------------------------------------------+
+| OME-XML, zipped           | Great – 124,764 bytes      | Worst – Entire dataset must be uncompressed, then the plane must be   |
+|                           |                            | decoded from base64.                                                  |
++---------------------------+----------------------------+-----------------------------------------------------------------------+
+| OME-XML, 7-zipped         | Great – 125,830 bytes      | Worst – Entire dataset must be uncompressed, then the plane must be   |
+|                           |                            | decoded from base64.                                                  |
++---------------------------+----------------------------+-----------------------------------------------------------------------+
+
+The performance penalty for accessing individual image planes from
+externally compressed formats (zip, gzip, and 7-zip) is high, since the
+data must be decompressed. There is little penalty for accessing them
```

Comment:
What specifically are the penalties here:

- decompression overhead (cpu, memory)
- storing uncompressed on disk
- disk bandwidth utilisation

etc..